external/fv3fit/fv3fit/keras/_models/models.py

```diff
@@ -305,51 +265,26 @@ def fit(
             validation_data = X_val[val_sample], y_val[val_sample]
         else:
             validation_data = None
+        self.training_loop.fit_loop(
+            self.model,
+            Xy=Xy,
+            validation_data=validation_data,
+            callbacks=[self._end_of_epoch_callback],
+        )
 
-        return self._fit_loop(Xy, validation_data, **fit_kwargs,)
-
-    def _fit_loop(
-        self,
-        Xy: Sequence[Tuple[np.ndarray, np.ndarray]],
-        validation_data: Optional[Tuple[np.ndarray, np.ndarray]],
-        epochs: int,
-        batch_size: Optional[int] = None,
-        workers: int = 1,
-        max_queue_size: int = 8,
-        use_last_batch_to_validate: bool = False,
```

Comment:
I guess these last two kwargs were not actually used here?

Reply:
That is correct.